src/modules/time.rs

```diff
@@ -0,0 +1,29 @@
+use ansi_term::Color;
+use chrono::{DateTime, Local};
+
+use super::{Context, Module};
+
+/// Outputs the current time
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("time")?;
+
+    // Remove when logic for disabled by default exists
+    if module.config_value_bool("disabled").unwrap_or(true) {
+        return None;
+    }
+
+    let is_12hr = module.config_value_bool("12hr").unwrap_or(false);
+    let default_format = if is_12hr { "%I:%M%p" } else { "%H:%M" };
+    let time_format = module
+        .config_value_str("format")
+        .unwrap_or(default_format)
+        .to_owned();
+
+    let local: DateTime<Local> = Local::now();
+
+    module.set_style(Color::Yellow.bold());
+    module.new_segment("time", &format!("[{}]", local.format(&time_format)));
```

Comment:
I don't know that we should have the brackets on either side of the module's output.
Perhaps we could have a `prefix` and `suffix` segment that are empty by default, but could be used to add these brackets?

```toml
[time]
prefix = "["
suffix = "]"

Reply:
Don't suffixes only trigger if there's a module after the current one? If time is the last thing displayed on a line, the prompt will look like

```
v1.3.6 at [10:35
```
which I think looks unbalanced.

Reply:
Only prefixes behave that way, not suffixes.
That said, these would be segments titled "prefix" and "suffix". They would not be instances of the `ModuleAffix` class, as those are usually colorless.

Sorry if that's confusing. 😅

Reply:
Oh, I see. This would allow the user to configure the symbols they want used to wrap the time using `starship.toml`

Makes sense!

Reply:
So ... I have made this change (https://github.com/starship/starship/pull/138/commits/f966b3dd1d28c0d9c0be65e6afa78b5ceb979b9b) ... but I just have one question, which is, should I still have this line?

```rust
module.get_prefix().set_value("");
```

or should I change it to

```rust
module.get_prefix().set_value("at");
```

Reply:
Nevermind ... just saw @matchai's review further on addressing my question 😄